packages/web-twig/src/Resources/components/Modal/README.md

```diff
@@ -1,12 +1,241 @@
 # Modal
 
+## Composed Modal (DEPRECATED)
+
 This is Twig implementation of the [Modal] component.
 
+Modal is a composition of several subcomponents:
+
+- [ModalComposed](#modalcomposed)
+  - [ModalDialog](#modaldialog)
+  - [ModalHeader](#modalheader)
+  - [ModalBody](#modalfooter)
+  - [ModalFooter](#modalfooter)
+
+⚠️ **DEPRECATED:** The component `ModalComposed` — which is currently necessary to
+distinguish the composed version of Modal from the simple one — is deprecated
+and will be removed in next major version: the composed Modal will supersede the
+[Simple Modal](#simple-modal-deprecated) variant.
+
+### ModalComposed
+
+#### Examples:
+
+```twig
+<ModalComposed id="modal-composed">
+  …
+</ModalComposed>
+<ModalComposed id="modal-composed" titleId="modal-composed-title">
+  …
+</ModalComposed>
+```
+
+👉 Please note the `titleId` attribute is linked to the title inside
+[Modal Header](#modalheader) and provides an accessible name for the dialog.
+
+#### API
+
+| Prop name | Type     | Default | Required | Description                        |
+| --------- | -------- | ------- | -------- | ---------------------------------- |
+| `class`   | `string` | `null`  | no       | Custom CSS class                   |
+| `id`      | `string` | —       | yes      | Modal ID                           |
+| `titleId` | `string` | `null`  | no       | ID of the title inside ModalHeader |
+
+On top of the API options, you can add `data-*` or `aria-*` attributes to
+further extend component's descriptiveness and accessibility. These attributes
+will be passed to the topmost HTML element of the component.
+
+### ModalDialog
+
+ModalDialog is the actual dialog window, a place for the header, body, and
```

Comment:
What about adding a link to the example with the form method dialog in the web package?

Reply:
It is slightly different in twig, so I have the implementation explained here too.